packages/handlebars-helper-attrs/README.md

```diff
@@ -0,0 +1,4 @@
+Handlebars Helper: attributes
```

Comment:
I thought we weren't going to specifically call out handlebars in the title/name of this helper since it can be applied to other languages as well.

Reply:
@elizabethsjudd 

I swear... `sindresorhus` is the "simpsons did it" of the node.js world.

https://github.com/sindresorhus/stringify-attributes

removed the code part which does what his module does (better), so this is indeed just a handlebars helper now...